### ISSUE TITLE
Vector path command parameters are missing for stickers

### ISSUE THREAD

**Issue by atiyka** [tdlib/td#2225]
I'm requesting the messages in a chat using `GetChatHistoryAsync`. One of the message's content is a sticker data. I can download the sticker and play the animation, but before that I'd like to display to the user the Outline vector. I receive the following object:

![image](https://user-images.githubusercontent.com/9460253/206455041-cc7f683d-6fdd-45a4-94f2-6d878287f17b.png)

So I get a list of `VectorPathCommand`, any of then has a DataType, for ex. `vectorPathCommandCubicBezierCurve`. But there are no parameters like `start_control_point` or `end_control_point` as the [documentations](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1vector_path_command_cubic_bezier_curve.html) says.

My question is that is this a bug in tdlib? Or I need to call another specific method to get the parameters? 
I'm using the tdlib in C#.

**Comment by atiyka** [comment]
Okay, and what is the way to convert the array entries?
But anyway as you can see from the screenshot in the response received directly from the tdlib, there are no control points at all.
Would it be possible that the tdlib has a conversion error when parsing the response from the telegram servers?

**Comment by levlam** [comment]
> that the tdlib has a conversion error when parsing the response from the telegram servers?

No. TDLib API is strictly typed, and can't have fields missing.

> Okay, and what is the way to convert the array entries?

You need to check wrapper documentation, or search for the definiton of `VectorPathCommandCubicBezierCurve` class.

**Comment by atiyka** [comment]
Thanks for the response.
I checked the tdlib code and I found that for creating the `Commands` array the `get_sticker_minithumbnail` method is used in `StickersManager:1933`.
The return value type is `vector<td_api::object_ptr<td_api::closedVectorPath>>` and indeed the values are set correctly. So I move forward to check the C# api.

However I see that there is commented code, which converts the commands to an svg string in `StickersManager:2123-2164`. Would it be possible to add this code and create a method which returns this information?
